examples/docker-compose.yaml

```diff
@@ -2,12 +2,12 @@ version: '2'
 
 services:
   owncast:
-    image: gabekangas/owncast:0.0.2
+    image: gabekangas/owncast:0.0.3
```

Comment:
I think I would change this to `owncast:latest` to avoid having to update this file with every release.

Reply:
Agree, done